WARNING | 2021-12-02 01:19:04,877 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_mh_context
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_maildir_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_imaps_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_c_tab
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_smtps_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_nntp_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_smtp_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_msgset_formats
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_pops_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_mh_record
WARNING | 2021-12-02 01:19:04,878 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_prog_record
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_strin
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_pop_record
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_mh_profile
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_program_name
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_sendmail_record
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_mbox_record
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_imap_record
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_strerr
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_strout
WARNING | 2021-12-02 01:19:04,879 | cle.backends.externs | Symbol was allocated without a known size; emulation may fail if it is used non-opaquely: mu_version_copyright
WARNING | 2021-12-02 01:19:04,887 | cle.loader | For more information about "Symbol was allocated without a known size", see https://docs.angr.io/extending-angr/environment#simdata
ERROR   | 2021-12-02 01:29:08,134 | root     | 
ERROR   | 2021-12-02 01:29:08,494 | root     | got an error while analyzing pick_yyparse
ERROR   | 2021-12-02 01:37:13,382 | root     | b'out of memory'
ERROR   | 2021-12-02 01:37:13,386 | root     | got an error while analyzing augment_argv
ERROR   | 2021-12-02 01:38:33,028 | root     | b'out of memory'
ERROR   | 2021-12-02 01:38:33,029 | root     | got an error while analyzing has_folder_option
ERROR   | 2021-12-02 01:39:54,988 | root     | b'out of memory'
ERROR   | 2021-12-02 01:39:54,992 | root     | got an error while analyzing mh_getopt_ext
ERROR   | 2021-12-02 01:42:07,645 | root     | b'out of memory'
ERROR   | 2021-12-02 01:42:07,653 | root     | got an error while analyzing part_match
ERROR   | 2021-12-02 01:42:15,179 | root     | b'out of memory'
ERROR   | 2021-12-02 01:42:15,190 | root     | got an error while analyzing match_char_class
ERROR   | 2021-12-02 01:43:55,765 | root     | b'out of memory'
ERROR   | 2021-12-02 01:43:55,766 | root     | got an error while analyzing mh_vgetyn
ERROR   | 2021-12-02 01:44:25,574 | root     | b'out of memory'
ERROR   | 2021-12-02 01:44:25,574 | root     | got an error while analyzing mh_expand_name
ERROR   | 2021-12-02 01:44:26,210 | root     | b'out of memory'
ERROR   | 2021-12-02 01:44:26,211 | root     | got an error while analyzing mh_find_file
ERROR   | 2021-12-02 01:48:26,504 | root     | b'out of memory'
ERROR   | 2021-12-02 01:48:26,507 | root     | got an error while analyzing mh_quote
ERROR   | 2021-12-02 01:50:31,212 | root     | b'out of memory'
ERROR   | 2021-12-02 01:50:31,215 | root     | got an error while analyzing yytnamerr
ERROR   | 2021-12-02 01:50:32,082 | root     | b'out of memory'
ERROR   | 2021-12-02 01:50:32,150 | root     | got an error while analyzing yysyntax_error
ERROR   | 2021-12-02 02:14:49,218 | root     | b'out of memory'
ERROR   | 2021-12-02 02:14:49,219 | root     | got an error while analyzing yylex_func
ERROR   | 2021-12-02 02:14:53,388 | root     | b'out of memory'
ERROR   | 2021-12-02 02:14:53,388 | root     | got an error while analyzing new_node
ERROR   | 2021-12-02 02:14:59,445 | root     | b'out of memory'
ERROR   | 2021-12-02 02:14:59,445 | root     | got an error while analyzing mh_format_dump_code
ERROR   | 2021-12-02 02:16:08,347 | root     | b'out of memory'
ERROR   | 2021-12-02 02:16:08,349 | root     | got an error while analyzing emit_funcall
ERROR   | 2021-12-02 02:16:13,953 | root     | b'out of memory'
ERROR   | 2021-12-02 02:16:13,958 | root     | got an error while analyzing codegen_node
ERROR   | 2021-12-02 02:19:28,989 | root     | list index out of range
ERROR   | 2021-12-02 02:19:28,989 | root     | got an error while analyzing parse_count
ERROR   | 2021-12-02 02:19:29,858 | root     | b'out of memory'
ERROR   | 2021-12-02 02:19:29,860 | root     | got an error while analyzing msgset_first
ERROR   | 2021-12-02 02:23:45,827 | root     | b'out of memory'
ERROR   | 2021-12-02 02:23:45,828 | root     | got an error while analyzing parse_range
ERROR   | 2021-12-02 02:36:39,109 | root     | b'out of memory'
ERROR   | 2021-12-02 02:36:39,109 | root     | got an error while analyzing ali_list_to_string
ERROR   | 2021-12-02 02:37:14,116 | root     | b'out of memory'
ERROR   | 2021-12-02 02:37:14,116 | root     | got an error while analyzing aliascmp
ERROR   | 2021-12-02 02:38:06,298 | root     | b'out of memory'
ERROR   | 2021-12-02 02:38:06,298 | root     | got an error while analyzing ali_yylex
ERROR   | 2021-12-02 02:38:06,937 | root     | b'out of memory'
ERROR   | 2021-12-02 02:38:06,938 | root     | got an error while analyzing yy_get_next_buffer
ERROR   | 2021-12-02 02:38:07,576 | root     | b'out of memory'
ERROR   | 2021-12-02 02:38:07,576 | root     | got an error while analyzing yy_get_previous_state
ERROR   | 2021-12-02 02:38:08,218 | root     | b'out of memory'
ERROR   | 2021-12-02 02:38:08,218 | root     | got an error while analyzing yy_try_NUL_trans
ERROR   | 2021-12-02 02:38:10,252 | root     | b'out of memory'
ERROR   | 2021-12-02 02:38:10,252 | root     | got an error while analyzing ali_yy_load_buffer_state
ERROR   | 2021-12-02 02:41:01,791 | root     | list index out of range
ERROR   | 2021-12-02 02:41:01,792 | root     | got an error while analyzing yy_fatal_error
ASSERTION VIOLATION
File: ../src/ast/shared_occs.cpp
Line: 119
UNEXPECTED CODE WAS REACHED.
Z3 4.8.13.0
Please file an issue with this message and more detail about how you encountered it at https://github.com/Z3Prover/z3/issues/new
